<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>IMAccountManager</class>
 <widget class="QWidget" name="IMAccountManager" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>@product@ - My Instant Messaging Accounts (MSN, Jabber...)</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTreeWidget" name="treeWidget" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="lineWidth" >
      <number>1</number>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Protocol</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Login</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="3" >
      <widget class="QPushButton" name="deleteIMAccountButton" >
       <property name="text" >
        <string>Delete</string>
       </property>
       <property name="icon" >
        <iconset resource="../../../../resources/wengophone.qrc" >:/pics/delete.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2" >
      <widget class="QPushButton" name="modifyIMAccountButton" >
       <property name="text" >
        <string>Modify</string>
       </property>
       <property name="icon" >
        <iconset resource="../../../../resources/wengophone.qrc" >:/pics/edit.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="4" >
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QPushButton" name="addIMAccountButton" >
       <property name="text" >
        <string>Add</string>
       </property>
       <property name="icon" >
        <iconset resource="../../../../resources/wengophone.qrc" >:/pics/add.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>treeWidget</tabstop>
  <tabstop>addIMAccountButton</tabstop>
  <tabstop>modifyIMAccountButton</tabstop>
  <tabstop>deleteIMAccountButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../resources/wengophone.qrc" />
 </resources>
 <connections/>
</ui>
